scripts/packagecloud_push.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -eo pipefail
+
+GITTAG=${GITHUB_REF#"refs/tags/"}
+
+if [ ! -n "${GITTAG}" -o ! -n "${PACKAGECLOUD_TOKEN}" -o ! -n "${PACKAGE_NAME}" ]; then
+  echo "GITTAG, PACKAGECLOUD_TOKEN and PACKAGE_NAME must be set"
+  exit 1
+fi
+
+gem install specific_install
+gem specific_install -l https://github.com/morpheu/fpm -b pleaserun_extra_options
+
+export PACKAGE_VERSION=${GITTAG}
+export PACKAGE_DIR="./dist/${PACKAGE_NAME}_linux_amd64"
+
+# NOTE: compatibility with goreleaser 1.8.3 and later
+# See more: https://github.com/goreleaser/goreleaser/commit/63436392db6ac0557513535fc3ee4223a44810ed
+if [[ -d "${PACKAGE_DIR}_v1" ]]; then
+  PACKAGE_DIR="${PACKAGE_DIR}_v1"
+fi
+
+if [[ ! -d "${PACKAGE_DIR}" ]]; then
+  PACKAGE_DIR="./dist/unpoller_linux_amd64"
+
+  if [[ ! -d ${PACKAGE_DIR} ]]; then
+    PACKAGE_DIR="${PACKAGE_DIR}_v1"
+  fi
+fi
+
+sudo apt-get update && sudo apt-get install rpm -y
+gem install package_cloud
+
+ruby misc/fpm_recipe.rb
+
+PACKAGE_CLOUD_REPO="golift/pkgs"
+if [[ ${PACKAGE_VERSION} =~ .+-rc ]]; then
+  PACKAGE_CLOUD_REPO="golift/unstable"
+fi
+
+SUPPORTED_UBUNTU_VERSIONS="focal"
+SUPPORTED_REDHAT_VERSIONS="7"
+SUPPORTED_DEBIAN_VERSIONS="buster"
```

Comment:
I don't do this either. Debian can use the focal repo too. My install.sh and repo.sh scripts both point all apt users to focal.